modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -160,6 +161,10 @@ public ObjectAction addObjectAction(
 		objectAction.setSystem(system);
 		objectAction.setStatus(ObjectActionConstants.STATUS_NEVER_RAN);
 
+		if ((description != null) && (description.length() > 75)) {
+			throw new ObjectActionDescriptionException.MustBe75Characters();
+		}
```

Comment:
@Aquiiiles please move this logic to a new method called `_validateDescription`.

You should call this new method after the `_validateExternalReferenceCode` and before the `_validateErrorMessage`